pyseir/models/suppression_policies.py

```diff
@@ -69,6 +69,92 @@ def generate_triggered_suppression_model(t_list, lockdown_days, open_days, reduc
     return interp1d(t_list, rho, fill_value='extrapolate')
 
 
+def generate_covidactnow_scenarios(t_list, R0, t0, scenario):
+    """
+    Generate a suppression policy for CovidActNow which generates an Reff on a
+    given date according to the policies in place.
+
+    Implements CovidActNow's version, which sets Reff
+        ```
+        def get_interventions(start_date=datetime.now().date()):
+            return [
+                None,  # No Intervention
+                {  # Flatten the Curve
+                    start_date: 1.3,
+                    start_date + timedelta(days=30) : 1.1,
+                    start_date + timedelta(days=60) : 0.8,
+                    start_date + timedelta(days=90) : None
+                },
+                {  # Full Containment
+                    start_date : 1.3,
+                    start_date + timedelta(days=7) : 0.3,
+                    start_date + timedelta(days=30 + 7) : 0.2,
+                    start_date + timedelta(days=30 + 2*7) : 0.1,
+                    start_date + timedelta(days=30 + 3*7) : 0.035,
+                    start_date + timedelta(days=30 + 4*7) : 0
+                },
+                {  # Social Distancing
+                    start_date: 1.7,
+                    start_date + timedelta(days=90) : None
+                },
+            ]
+        ```
+
+    Returns
+    -------
+    suppression_model: callable
+        suppression_model(t) returns the current suppression model at time t.
+    """
+    # Rho is the multiplier on the contact rate. i.e. 1 = no change, 0 = no transmission
+    rho = []
+    for t in t_list:
+        actual_date = t0 + timedelta(days=t)
+        today = datetime.utcnow()
+
+        if scenario == 'no_intervention':
+            rho.append(1)
+
+        elif scenario == 'flatten_the_curve':
+            if actual_date <= today:
+                rho.append(1)
+            elif (actual_date - today).days <= 30:
+                rho.append(1.3 / R0)
```

Comment:
Are rhos values selected by covidactnow? I am confused how this and the date together translates R_eff.

Reply:
Our suppression policy is usually a number between 0 - 1. They hardcoded R_eff rather than XX percent reduction. This fixes that mapping to reproduce an R_eff after intervention.